charts/rancher-gke-operator/1.0.100/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+name: rancher-gke-operator
+description: A Helm chart for provisioning GKE clusters
+home: https://github.com/rancher/gke-operator
+sources:
+  - "https://github.com/rancher/gke-operator"
+version: 1.0.100
+appVersion: v1.0.1-rc13
```

Comment:
I can't comment on the old thread, but we can't merge this until we release GKE image. GKE is not finished yet.